scripts/vmbuild.rb

```diff
@@ -152,7 +139,7 @@ def verify_run(output)
     FileUtils.mkdir_p(destination_directory)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{manageiq_checkout.commit_sha}.#{target.file_extension}"
     destination = destination_directory.join(file_name)
-    $log.info `mv #{source} #{destination}`
+    $log.info `ln #{source} #{destination}`
```

Comment:
I need the imagefactory image to continue to exist in /build/storage, so that I can imagefactory to publish the image in a separate step (becomes moot if we collapse the steps). I'm using a hard link instead of a copy so that I can more efficiently find back the image UUID from the image in /build/fileshare.